### PR TITLE
create header and render dummy cards and some layout

### PR DESCRIPTION
**Pull request by muhammed-basaran**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,3 +1,13 @@
+import Header from "../components/Header";
+import Card from "../components/Card";
+import styled from "styled-components";
+
 export default function Home() {
-  return <h1>Hello Beavers 🦫</h1>;
+  return (
+    <>
+      <Header />
+
```
Comment (julialoeschel) [inline pages/index.js]: no blank spaces, please :)

File: components/Card.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Itemlist from "./Itemlist";
+import styled from "styled-components";
+
+export default function Card() {
+  return (
+    <>
+      <StyledCard>
```
Comment (julialoeschel) [inline components/Card.js]: The naming is irritating. 
I am expecting a Card here (single, one shoe). 
But I do get a Container for a lot of items  (please use a List HTML element for that)

File: components/Itemlist.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import { shoes } from "../public/shoes";
+import styled from "styled-components";
+
+export default function Itemlist() {
+  return (
+    <>
```
Comment (julialoeschel) [inline components/Itemlist.js:6]: all your Listitems should be in a List (see above) and therefore, this should be a Listitem <li> 
This is actually a single Card, right? - check the naming.

File: components/Itemlist.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      {shoes.map((shoe) => (
+        <StyledItemList>
```
Comment (julialoeschel) [inline components/Itemlist.js]: You call it ItemList and it is a Section. I think its more of a container.

File: components/Itemlist.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+          <StyledImage src={shoe.img} width="50%" height="70%" />
+          <StyledListContainer>
```
Comment (julialoeschel) [inline components/Itemlist.js]: You are using a 'ul' here - with only one ListItem. 
you might not need the 'ul ' here.

File: components/Itemlist.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+const StyledDivContainer = styled.div`
+  padding: 15px;
+  margin: auto;
+  margin-left: 20px;
+  width: 90%;
+  height: 20%;
+`;
+const StyledItemDate = styled.span`
+  font-size: 16px;
+
```
Comment (julialoeschel) [inline components/Itemlist.js]: no blank spaces :)

**Review by julialoeschel — COMMENTED** [review]
Hey Mo. 
good work. 

I left some Comments for you, the Structure needs a little more love :) 

other than that - good job! 🐜

File: pages/index.js (hunk 1 above)
Comment (muhammed-basaran) [reply pages/index.js]: the blank spaces are coming automatically by save the file

File: components/Itemlist.js (hunk 5 above)
Comment (muhammed-basaran) [reply components/Itemlist.js]: okay i delete the ul and change it with a h2

**Pull request merged**